src/shims/shims.proj

```diff
@@ -74,8 +74,8 @@
     <!-- Copy the facades to the package ref and lib folders to be included in the packages -->
     <!-- TODO: replace with BinPlacing targets -->
     <ItemGroup>
-      <PackageOutputPaths Include="$(BinDir)$(TargetGroup)/pkg/ref" />
-      <PackageOutputPaths Include="$(BinDir)$(TargetGroup)/pkg/lib" />
+      <PackageOutputPaths Include="$(BinDir)pkg/$(TargetGroup)/ref" />
```

Comment:
I realize these will be correct but I would still prefer we share a common path from dir.props here instead of maintaining the pkg output directory structure in multiple locations.